packages/dev/sharedUiComponents/src/fluent/hoc/fileUploadLine.tsx

```diff
@@ -1,9 +1,10 @@
 import { useRef } from "react";
 import type { FunctionComponent } from "react";
 import { ButtonLine } from "./buttonLine";
-import type { ButtonLineProps } from "./buttonLine";
+import type { ButtonProps } from "./buttonLine";
+import { DocumentAdd16Regular } from "@fluentui/react-icons";

```

Comment:
So far I've been only using the unsized icons (e.g. `DocumentAddRegular`). These can all be sized through fontSize without increasing size by having the same icon of different fixed sizes.